packages/enhanced/src/lib/container/runtime/RemoteEntryPlugin.ts

```diff
@@ -0,0 +1,30 @@
+import type { Compiler, WebpackPluginInstance } from 'webpack';
+import pBtoa from 'btoa';
+
+export class RemoteEntryPlugin implements WebpackPluginInstance {
+  readonly name = 'VmokRemoteEntryPlugin';
+  private _name: string;
+  private _getPublicPath: string;
+
+  constructor(name: string, getPublicPath: string) {
+    this._name = name;
+    this._getPublicPath = getPublicPath;
+  }
+
+  apply(compiler: Compiler): void {
+    if (!this._getPublicPath.startsWith('function')) {
```

Comment:
@2heal1 @zhoushaw i can replace error throw with original implementation. if its not a function, i can wrap the string as the function body and use new Function?

Reply:
It's better this way. It's compatible with the old logic

Reply:
okay will ammend